manifests/init.pp

```diff
@@ -15,7 +15,10 @@
 # $version::                    foreman package version, it's passed to ensure parameter of package resource
 #                               can be set to specific version number, 'latest', 'present' etc.
 #
-# $port::                       Port on which will foreman proxy listen
+# $http_port::                  HTTP port to listen on (if http is enabled)
+#                               type:integer
+#
+# $ssl_port::                   SSL port to listen on (if ssl is enabled)
```

Comment:
Should this be https_port? Technically I think it's served over TLS rather than SSL.